tasks/src/app/simple-button/simple-button.component.ts

```diff
@@ -1,22 +1,40 @@
-import { Component, OnInit } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
 
 @Component({
 	selector: 'simple-button',
 	templateUrl: './simple-button.component.html',
-	styleUrls: ['./simple-button.component.css']
+	styleUrls: ['./simple-button.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SimpleButtonComponent implements OnInit {
-	public numberOfClicks: number;
+	public inputStatus: boolean;
```

Comment:
```suggestion
	public isInputEnable: boolean;
```
for boolean variable use is/are/has before. It is more obvious that you have a boolean
rename also the function that changes this variable